tox.ini

```
@@ -59,7 +59,7 @@ envlist =
     dogpile_contrib-py{36,37,38,39}-dogpilecache{06,07,08,09,10,}
     falcon_contrib{,_autopatch}-{py27,py35,py36,py37}-falcon{10,11,12,13,14}
     falcon_contrib{,_autopatch}-{py35,py36,py37}-falcon{20}
-    flask_contrib{,_autopatch}-{py27,py35,py36}-flask{010,011,012,10}-blinker-pytest3
+    flask_contrib{,_autopatch}-{py27,py35,py36,py37,py38,py39}-flask{010,011,012,10,11}-blinker-pytest3
```

should we also include "latest" here?
```suggestion
    flask_contrib{,_autopatch}-{py27,py35,py36,py37,py38,py39}-flask{010,011,012,10,11,}-blinker-pytest3
```

and then:

```diff
+ flask: flask>=1.1
```

---

wfm